Input/input.css

```diff
@@ -0,0 +1,37 @@
+@import '../style.css';
+
+.float-container {
+    border: solid 2px var(--shadow-color);
+    position: relative;
+    padding: 0 8px;
+    border-radius: 10px;
+  }
+  .float-container input {
+    border: none;
+    font-size: 1rem;
+    outline: 0;
+    padding: 16px 0 10px;
+  }
+
+  .input-lbl {
+    font-size: 0.75rem;
+    position: absolute;
+    margin-top: 2px;
+    color: var(--shadow-color);
```

Comment:
You can change the color of label so that it could be visible.

Reply:
i don't think so it is necessary !